noghartt/noctis
# feat(users): write authentication module and logic

## Overview

With this PR, you can login and get the return as a token (JWT based).

## How to test

After you run `createUser` mutation (on #4, you can see how test and create a user), run this mutation:

```graphql
mutation login($email: String!, $password: String!) {
  login(email: $email, password: $password)
}
```
